include/shards/shards.swift

```diff
@@ -11,8 +11,8 @@ public struct Globals {
 
     init() {
         Core = shardsInterface(UInt32(SHARDS_CURRENT_ABI))
-        print("Shards Swift runtime initialized!")
-        Core.pointee.registerShard(BaseShard.name.withUnsafeBufferPointer{ $0.baseAddress }, { createSwiftShard(BaseShard.self) })
+        // print("Shards Swift runtime initialized!")
```

Comment:
Clean up comment

Reply:
Oops this should be in other pr as well 😅. But it's simple I'll wrap it here